slurm.sh

```diff
@@ -135,6 +138,18 @@ slurm_job_status_from_sacct() {
     return $jobExitCode
 }
 
+# Get slurm job resource usage and efficiency statistics
+
+slurm_resource_usage_summary(){
+    local jobId=$1
+
+    check_variables 'jobId'
+
+    local jobInfo="$(seff $jobId 2> /dev/null )"
```

Comment:
`2> /dev/null` suppresses any error messages (stderr). Redirecting stderr to /dev/null means any errors from `slurm_resource_usage_summary` will be completely hidden. If the function/command fails, $summaryStats will be empty, and the user won't know why.

I have added a check in `atlas-slurm`, but feel free to bring the check here.

Reply:
The redirection of stderr to /dev/null here specifically is to remove the warning`perl: warning: Setting locale failed.` and all lines following it. But no harm in _not_ suppressing it, and I agree that we should be able to see other potential errors despite checks in place prior to the command. I'm removing it.

Reply:
Done in 960a2ec39458750bb185f3f55b9e2f8af959e920